caldp/sysexit.py

```diff
@@ -286,6 +295,14 @@ def exit_receiver():
     except MemoryError:
         code = exit_codes.CALDP_MEMORY_ERROR
         _report_exception(code, ("Untrapped memory exception.",))
+    except OSError as exc:
```

Comment:
_technically_ OSError is also thrown for I/O problems like missing or full disks. It's probably ok to rescue them up the ladder, but I can imagine a scenario where EBS goes down and all jobs fail with an OSError, then get rescued all the way up the ladder to 64g or something. Just a thought experiment.

Reply:
My intent for this is that if "Cannot allocate memory" is part of the OSError's str() or repr(),  then issue an OS_MEMORY_ERROR exit code.   Otherwise do what it always did for OSError.

Reply:
My mistake, I commented before really reading the entire handling of the exception. I saw the simplicity of the above exception being handled and just totally spaced it and went off looking for exactly what throws OSError's.